python/kserve/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
@spolti Let's use poetry 1.8?

Reply:
sure, let me update my end.

Done. Thanks for the heads up.